internify/src/components/molecules/TechStack.js

```diff
@@ -1,80 +1,224 @@
+          label={"C"}
+          onChange={handleLanguageChange}
+          checked={langState.C}
+          />
         </div>
         <TextField id="languages" label="Custom..." variant="filled" />
       </div>
       <div className="tech_stack_container">
         <h3>Frameworks*</h3>
         <div className="tech_stack_checkbox_container">
-          {frameworks.map((x) => {
-            return (
-              <CheckBox key={x} name={x} label={x} onChange={handleFrameworksChange} />
-            );
-          })}
+          <CheckBox 
+          key={"React"}
+          name={"React"}
+          label={"React"}
+          onChange={handleFrameworksChange}
+          checked={frameState.React}
+          />
+          <CheckBox 
+          key={"Angular"}
+          name={"Angular"}
+          label={"Angular"}
+          onChange={handleFrameworksChange}
+          checked={frameState.Angular}
+          />
+          <CheckBox 
+          key={"HTML"}
+          name={"HTML"}
+          label={"HTML"}
+          onChange={handleFrameworksChange}
+          checked={frameState.HTML}
+          />
+          <CheckBox 
+          key={"CSS"}
+          name={"CSS"}
+          label={"CSS"}
+          onChange={handleFrameworksChange}
+          checked={frameState.CSS}
+          />
         </div>
         <TextField id="languages" label="Custom..." variant="filled" />
       </div>
       <div className="tech_stack_container">
         <h3>Work Tools and Others*</h3>
         <div className="tech_stack_checkbox_container">
-          {workTools.map((x) => {
-            return (
-              <CheckBox key={x} name={x} label={x} onChange={handleWorkToolsChange} />
-            );
-          })}
+          <CheckBox 
+          key={"Jira"}
+          name={"Jira"}
+          label={"Jira"}
+          onChange={handleWorkToolsChange}
+          checked={toolsState.Jira}
+          />
+          <CheckBox 
+          key={"Asana"}
+          name={"Asana"}
+          label={"Asana"}
+          onChange={handleWorkToolsChange}
+          checked={toolsState.Asana}
+          />
+          <CheckBox 
+          key={"Confluence"}
+          name={"Confluence"}
+          label={"Confluence"}
+          onChange={handleWorkToolsChange}
+          checked={toolsState.Confluence}
+          />
+          <CheckBox 
+          key={"Notion"}
+          name={"Notion"}
+          label={"Notion"}
+          onChange={handleWorkToolsChange}
+          checked={toolsState.Notion}
+          />
         </div>
         <TextField id="languages" label="Custom..." variant="filled" />
       </div>
       <div className="tech_stack_container">
         <h3>Computer Science Concepts</h3>
         <div className="tech_stack_checkbox_container">
-          {csConcepts.map((x) => {
-            return (
-              <CheckBox key={x} name={x} label={x} onChange={handleConceptsChange} />
-            );
-          })}
+          <CheckBox 
```

Comment:
can we try to figure out a way to keep the old format and still update the state? checked={conceptState.x}?